fastlane/helper/plugin_scores_helper.rb

```diff
@@ -325,7 +326,13 @@ def parse_file(file, debug_state: false)
               end
             when :in_method
               if (string_statement = string_statement_from_line(line))
-                last_string_statement = string_statement
+                # Multiline support for `description` method
+                if last_method_name == 'description'
+                  # rubocop:disable BlockNesting
+                  last_string_statement.concat(string_statement) unless last_string_statement.nil?
```

Comment:
Won't `last_string_statement` always be `nil` here? 🤔 It gets `nil`-ed on line 309 and 344.

Reply:
Good question 👍 

It gets `nil`-ed on the end of the line. The regex to detect the end of the line looks like this:

```
def is_end?(line)
    !!line.match(/^\s*end\s*/)
end
```

So if we have a multi-line description, the end of the line is not detected and `last_string_statement` is not nil then 😉